packages/frontend/core/src/components/hooks/affine/use-enable-cloud.tsx

```diff
@@ -42,7 +46,8 @@ export const useEnableCloud = () => {
         if (!account) return;
         const { id: newId } = await workspacesService.transformLocalToCloud(
           ws,
-          account.id
+          account.id,
+          'affine-cloud'
```

Comment:
is this right?

Reply:
If no additional servers are added, should there only be affine-cloud? 
Currently, the workspace flavor only has local, affine-cloud, and "serverID".
If the default server base url is modified, it should still be affine-cloud by default, right?🧐